src/Uno.UWP/UI/Core/CorePhysicalKeyStatus.cs

```diff
@@ -10,5 +12,28 @@ public partial struct CorePhysicalKeyStatus
 		public bool IsMenuKeyDown;
 		public bool WasKeyDown;
 		public bool IsKeyReleased;
+
+		public bool Equals(CorePhysicalKeyStatus other) =>
+			RepeatCount == other.RepeatCount && ScanCode == other.ScanCode &&
+			IsExtendedKey == other.IsExtendedKey && IsMenuKeyDown == other.IsMenuKeyDown &&
+			WasKeyDown == other.WasKeyDown && IsKeyReleased == other.IsKeyReleased;
+
```

Comment:
Maybe we shouldn't add this new public API and only keep the `object.Equals` override? Not sure.